docs/04_For_Developers/06_Project_Maintainer_Guide.md

```diff
@@ -0,0 +1,58 @@
+# Project Maintainer Guide
+RSS-Bridge project maintainers are responsible for:
+1. Tagging issues
+1. Reviewing and merging pull requests
+1. Creating releases
+
+Project maintainers should keep all communications related to the project open
+to the community (Preferably GitHub issues but IRC/Matrix is also allowed).
+
+## Issues
+Maintainers should triage and respond to issues if possible, making sure to
+specify the tag if possible. For example, the 'Bug Report' can be clarified to
+'Bridge-Broken' once that is established. Project maintainers are encouraged to
+mention the bridge maintainer to comment on issues related to a specific bridge.
+
+An issue should remain open until it is resolved by fixing the issue or adding
+the feature it describes. There are some exceptions:
+- In the case of a feature request that will not be added, the issue can be
+  closed with the 'wontfix' label after pinging other active maintainers.
+- If an issue is too obscure or unclear and the issue creator does not respond
+  to additional clarification questions for 1 month, these issues should be
+  closed as they cannot be acted upon.
+
+## Pull Requests
+Maintainers should thoroughly **review** and **test** pull requests before
+merging them. Most bridge authors use other bridges in the project for reference
+so it is important to maintain a high quality codebase so that it does not
```

Comment:
I'm a bit conflicted on this point.I think a code review + merge without testing is fine. It lessens the maintenance burden.

Also I don't want to be too strict with PRs because I don't want to discourage contributors.

I think the most imortant stuff in a bridge with regards to maintainability is a good initial bridge form configuration (`PARAMETERS`). Because everything else can be refactored later.